mod/web_services/classes/ElggApiKey.php

```diff
@@ -119,4 +119,36 @@ public function regenerateKeys() {
 		
 		return true;
 	}
+	
+	/**
+	 * Check is the API key is active
+	 *
+	 * @return bool
+	 */
+	public function isKeyActive() {
+		$keys = $this->getKeys();
+		if (empty($keys)) {
+			return false;
+		}
+		
+		return (bool) $keys->active;
+	}
+	
+	/**
+	 * Enables the API key for use by API requests
+	 *
+	 * @return bool
+	 */
+	public function enableKey() {
+		return _elgg_services()->apiUsersTable->enableAPIUser($this->getPublicKey());
+	}
+	
+	/**
+	 * Disables the API key for use by API requests
+	 *
+	 * @return bool
+	 */
+	public function disableKey() {
```

Comment:
change function name to `disableKeys` to keep plurar usage of the keys word intact